hack/errata.py

```diff
@@ -44,7 +44,10 @@ def run(poll_period=datetime.timedelta(seconds=3600),
         for message in poll(period=2*poll_period, **kwargs):
             if cache and message['fulladvisory'] in cache or 'bug fix update' not in message['synopsis']:
                 continue
-            message['approved_pr'] = lgtm_fast_pr_for_errata(githubrepo, githubtoken, message)
+            try:
+                message['approved_pr'] = lgtm_fast_pr_for_errata(githubrepo, githubtoken, message)
+            except Exception as error:
+                _LOGGER.error('Error looking up PRs: {}'.format(error))
```

Comment:
nit: As a non-fatal complaint that might be due to handling some unrelated PR, this should be `_LOGGER.warn`.

Reply:
Updated, thanks